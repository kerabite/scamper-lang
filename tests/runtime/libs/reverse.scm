(reverse (list 1 2 3 4 5))

(reverse (list))

(reverse (reverse (list 1 2 3 4 5 6 7 8 9 10)))