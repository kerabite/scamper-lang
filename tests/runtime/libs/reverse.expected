(list 5 4 3 2 1)
null
(list 1 2 3 4 5 6 7 8 9 10)
